taf/tests/test_updater/test_clone/test_clone_invalid.py

```diff
@@ -111,11 +112,13 @@ def test_clone_invalid_target_invalid_metadata(origin_auth_repo, client_dir):
     setup_manager.add_task(swap_last_two_commits)
     setup_manager.execute_tasks()
 
+    # On CI, the error message is not always the same, but always reports that the metadata is invalid
+    # so use a more generic error pattern for now
     update_invalid_repos_and_check_if_repos_exist(
         OperationType.CLONE,
         origin_auth_repo,
         client_dir,
-        INVALID_VERSION_NUMBER_PATTERN,
```

Comment:
question: `INVALID_VERSION_NUMBER_PATTERN` is now an unused global variable. Is that intended? If so, let's check if any other variables are unused and remove them